### DISCUSSION TITLE
Setting chrome.exe Path for ChromiumWebBrowser

### DISCUSSION THREAD

**Discussion by Sehlor** [cefsharp/CefSharp#4598] [General]
Hello,

I want to set chrome.exe path and distribute chromium with my project and cef to use that chrome instead of already installed on users computer, to make sure my application works when user updates chrome.

I tried setting BrowserSubprocessPath but, it gives me error that says: 

network_service_instance_impl.cc(625)] Network service crashed, restarting service.

I am stuck, please help...

**Comment by amaitland** [comment]
Chrome is not designed to be directly embedded in 3rd party applications.  What you are attempting is not possible.

Each version of CefSharp is self contained, the relevant version of the chromium embedded framework is included. 

You can read more about the project at https://github.com/cefsharp/CefSharp/wiki/General-Usage
